### PR TITLE
Civix upgrade, replace deprecated debug command, copy legacycode class into extension to avoid dependency 

### LINKED ISSUE #45: Error due to removed Core method
#34 introduced some code that calls `CRM_Core_Form_Date::buildDateRange()` which got removed with https://github.com/civicrm/civicrm-core/commit/83966bf9111f2b7a5f33ec7716eca0c412d027dd in CiviCRM 5.61.0.

@pfigel you were the author of #34, would you be able to look into that?

### PR DESCRIPTION
**Pull request by jofranz**
_internal reference: 21474_

### CODE REVIEW

**Comment by jofranz** [comment]
@jensschuppe Please have a look. Sorry for the all-in-one branch approach. We should aim to avoid this.

File: CRM/Fastactivity/Legacycode/Date.php
Comment (jensschuppe) [inline CRM/Fastactivity/Legacycode/Date.php:1]: Let's name this class `CRM_Fastactivity_Legacycode_Form_Date` to resemble the original class name.

**Review by jensschuppe — CHANGES_REQUESTED** [review]

**Comment by jensschuppe** [comment]
Just for reference, https://github.com/systopia/de.systopia.donrec/pull/181 tries to actually replace calls to the `buildDateRange()` method. This should eventually be done here, as well.

File: CRM/Fastactivity/Legacycode/Date.php
Comment (jofranz) [reply CRM/Fastactivity/Legacycode/Date.php:1]: Would you suggest also placing it into a folder?:
`/Legacycode/Form/Date.php`

**Comment by jofranz** [comment]
> Just for reference, [systopia/de.systopia.donrec#181](https://github.com/systopia/de.systopia.donrec/pull/181) tries to actually replace calls to the `buildDateRange()` method. This should eventually be done here, as well.

I moved it into #47

File: CRM/Fastactivity/Legacycode/Date.php
Comment (jensschuppe) [reply CRM/Fastactivity/Legacycode/Date.php:1]: It actually has to, as otherwise the class loader would be unable to find the class.

**Pull request merged**